examples/machine-controller.yaml

```diff
@@ -98,7 +98,7 @@ rules:
 apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: Role
 metadata:
-  name: machine-controller
+  name: configmap-reader-kube-public
```

Comment:
I really dislike putting meta infos and content of something into its name instead of simply the purpose because:

* It is redundant
* It means the name has to be changed everytime the contents of that object change
* It is not not easily possible to find out afterwards what for this was created